uuid_cache_chip.csv

```diff
@@ -251,14 +680,3 @@ pkg-resj11569~(4527)-text-name-density~a,6c3385ba-a156-4faf-a882-90620f1a3e2d
 pkg-resj11569~(4527)-text-name-density~b,3c8a0bdf-d887-4b34-be1f-894f01440afc
 pkg-resj11569~(4527)-text-value-density~a,acbebc0b-57d8-4a0c-b9c8-57b6e7d603dd
 pkg-resj11569~(4527)-text-value-density~b,23fdb5ba-e441-445a-9ab3-95f035356501
-dev-resistor~0402~(01005)-dev,58678317-f386-416d-890a-470f59ab3fba
-dev-resistor~0603~(0201)-dev,324de04a-f2ac-4134-b8bf-914173c150bf
-dev-resistor~1005~(0402)-dev,5ff4a2e5-3593-49ab-bdbe-f9b68015a3ed
-dev-resistor~1608~(0603)-dev,483a71eb-318e-448e-82ff-f02efc4821aa
-dev-resistor~2012~(0805)-dev,078650d3-483c-4b9e-a848-b14f1aad2edc
-dev-resistor~3216~(1206)-dev,a6a6744d-7d3b-450a-b782-feca43939ca5
-dev-resistor~3225~(1210)-dev,a5153d16-a78b-439c-876b-12d12a607237
-dev-resistor~3246~(1218)-dev,3a5ce2ef-4c16-459f-a6b8-f914508a17a1
-dev-resistor~5025~(2010)-dev,5512ed40-604c-43fd-802c-1da10d419357
-dev-resistor~6432~(2512)-dev,52876458-2396-4ade-8280-6a033efa911f
-dev-resistor~11569~(4527)-dev,3cf91223-569c-4cf5-b2ec-64fa283bd04f
```

Comment:
What happened with these?

Reply:
They moved up, the file wasn't properly sorted before.